### ISSUE TITLE
MySQL full-copy - ddl - table partitioning - partition by linear key cause src task error

### ISSUE THREAD

**Issue by asiroliu** [actiontech/dtle#802]
## Description
MySQL full-copy - ddl - table partitioning - partition by linear key cause src task error

## Steps to reproduce the issue
1.  create table
```sql
SQL> CREATE TABLE t4 (col1 INT, col2 CHAR(5), col3 DATE) PARTITION BY LINEAR KEY(col3) PARTITIONS 5;
```
2. create dtle job

## Describe the results you received
src dtle log
```
2021-11-09T13:07:48.591+0800 [WARN]  client.driver_mgr.dtle: No valid unique key found. It will be slow on large table.: driver=dtle job=aaa-migration nKey=0 schema=test table=t4 @module=inspector timestamp=2021-11-09T13:07:48.590+0800
2021-11-09T13:07:48.592+0800 [ERROR] client.driver_mgr.dtle: error at ParseCreateTableStmt.: driver=dtle @module=dtle.extractor err="line 7 column 16 near "KEY (col3)
PARTITIONS 5 */" " job=aaa-migration timestamp=2021-11-09T13:07:48.592+0800
2021-11-09T13:07:48.592+0800 [INFO]  client.driver_mgr.dtle: Step: committing transaction: driver=dtle @module=dtle.extractor job=aaa-migration n=2 timestamp=2021-11-09T13:07:48.592+0800
2021-11-09T13:07:48.592+0800 [ERROR] client.driver_mgr.dtle: onError: driver=dtle job=aaa-migration @module=dtle.extractor err="line 7 column 16 near "KEY (col3)
PARTITIONS 5 */" " timestamp=2021-11-09T13:07:48.592+0800
2021-11-09T13:07:48.592+0800 [DEBUG] client.driver_mgr.dtle: extractor shutdown: driver=dtle @module=dtle.extractor job=aaa-migration timestamp=2021-11-09T13:07:48.592+0800
2021-11-09T13:07:48.592+0800 [INFO]  client.driver_mgr.dtle: extractor shutdown: driver=dtle @module=dtle.extractor job=aaa-migration timestamp=2021-11-09T13:07:48.592+0800
2021-11-09T13:07:48.593+0800 [INFO]  client.driver_mgr.dtle: Shutting down: driver=dtle @module=dtle.extractor job=aaa-migration timestamp=2021-11-09T13:07:48.593+0800
2021-11-09T13:07:48.594+0800 [INFO]  client.driver_mgr.dtle: DestroyTask: driver=dtle @module=dtle id=5e2b898f-f4a2-f301-88c7-fe20b62ec2cc/src/809e0c0f timestamp=2021-11-09T13:07:48.593+0800
2021-11-09T13:07:48.594+0800 [DEBUG] client.driver_mgr.dtle: extractor shutdown: driver=dtle @module=dtle.extractor job=aaa-migration timestamp=2021-11-09T13:07:48.594+0800
```

## Describe the results you expected
the table should copy to the dest

## Output of `./dtle version`:**

```
9.9.9.9-master-89033ce
```

## Additional information
src mysql show
```sql
mysql> show create table t4\G
*************************** 1. row ***************************
       Table: t4
Create Table: CREATE TABLE `t4` (
  `col1` int(11) DEFAULT NULL,
  `col2` char(5) DEFAULT NULL,
  `col3` date DEFAULT NULL
) ENGINE=InnoDB DEFAULT CHARSET=latin1
/*!50100 PARTITION BY LINEAR KEY (col3)
PARTITIONS 5 */
1 row in set (0.00 sec)
```

**Comment by ffffwh** [comment]
fixed after 232ff9ce.

**Comment by asiroliu** [comment]
verified version:
9.9.9.9-master-aa4de8d
